lib/notify.js

```diff
@@ -2,39 +2,44 @@ var Handlebars    = require('handlebars')
 var request       = require('request')
 var Logger        = require('./logger')
 
-module.exports.notify = function(metadata, config) {
-
-  if (config.notify && config.notify.content) {
+var handleNotify = function(metadata, notifyEntry) {
+  var regex
+  if(metadata.name && notifyEntry.packagePattern) {
```

Comment:
Could you standardize code styling? eg: `if (` instead `if(`. The same  below for `for` below. In part is my fault because `eslint` strict styling hasn't been worked out yet.